rest-service/manager_rest/security/authentication.py

```diff
@@ -31,6 +31,8 @@
 
 
 Authorization = namedtuple('Authorization', 'username password')
+Failed_auth_message = 'Authentication failed for <User username=`{0}`>. ' \
```

Comment:
Is there a reason variable starts with capital?

Reply:
nope. i saw the one above it and followed suit. Change to lower case?